app/src/main/java/hu/vmiklos/plees_tracker/Preferences.kt

```diff
@@ -20,6 +30,76 @@ class Preferences : PreferenceFragmentCompat() {
             it.summary = path
         }
     }
+
+    override fun onPreferenceTreeClick(preference: Preference): Boolean {
```

Comment:
We already have other settings that want to do something custom after being enabled, for example when you enable automatic backup. In that case we have a listener in SharedPreferencesChangeListener.kt to perform the custom action. Would you mind moving this logic to there as well, for consistency?

Reply:
Go for it. I wasn't really sure where to put this code to be honest but yeah for consistency it should be moved there.